src/core/create_compilers/RollupCompiler.ts

```diff
@@ -148,8 +148,12 @@ export default class RollupCompiler {
 			}
 		});
 
-		const resp = await bundle.generate({ format: 'cjs' });
-		const { code } = resp.output ? resp.output[0] : resp;
+		const {
```

Comment:
Is this now making a new assumption about what version of Rollup you're using? E.g., does this now only work with Rollup >=1? If so, this would be a breaking change.

Reply:
Good thing to check. This still works with Rollup 1 though. I just tested against the realworld sample app rewound back a few commits to when it was using Rollup 1.15.1

Reply:
I think I would consider this a breaking change though if this now only works with Rollup 1+. I don't know that the required peer dependency is officially documented anywhere - but this code was specifically here to support both < 1 and >= 1 (added by me in 7726325b4b037aaa8425f5d4e1c2fd6204b81658).

This seems like a reasonable breaking change to make, however, as Rollup 1 is now relatively old. If so, this should probably be lumped together with some other less-jarring but still-breaking changes.

Reply:
Oh, I missed that you said `>= 1` and just assumed you meant `> 1` since 0.x is so old now

So I went back to https://github.com/sveltejs/realworld/commit/0c1c67215e1a0a7a77458d70f4e6ef4270f43b45 and ran it as-is without this change and it worked, but it was relying on an old version of Sapper. Then I upgraded it to the latest version of Sapper (still without this change) and got `Error: Cannot find module 'svelte/compiler'`, so I think we may have broken Rollup 1 support awhile ago

In either case, I'd be a proponent of merging [the other breaking changes that are pending](https://github.com/sveltejs/sapper/pulls?q=is%3Aopen+is%3Apr+label%3Abreaking) and making the next release 0.28.0